choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -176,4 +222,21 @@ public void addAutoRoutine(String name, AutoRoutineGenerator generator) {
   public AutoRoutine getSelectedAutoRoutine() {
     return lastAutoRoutine;
   }
+
+  /**
+   * Gets a Command that runs the latest selected auto routine,
+   * stopping when the selected routine finishes.
+   * This Command can directly be bound to a trigger, like so:
+   * <pre><code>
+   *     AutoChooser chooser = ...;
+   *
+   *     public Robot() {
+   *         autonomous().whileTrue(chooser.autoCmd());
```

Comment:
```suggestion
   *         RobotModeTriggers.autonomous().whileTrue(chooser.autoCmd());
```

Reply:
Is it recommended to use the RobotModeTriggers namespace? Everywhere i see the trigger, i see it as a static imported method(if it's consistent with the wpilib api, then ill commit the suggestion)

Reply:
People wont know where this is from if you dont have this here